src/main/java/com/school/project/repository/UserAccountRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.school.project.repository;
+
+import com.school.project.model.UserAccount;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+public interface UserAccountRepository extends JpaRepository<UserAccount, Long> {
+    @Transactional
+    void deleteUserAccountByUser_Id(Long id);
```

Comment:
please, never use underscore in method names. Follow the camelCase name convention. (FYI it should work without underscore also)

Reply:
oh sorry,i forgot to change